File: EldenRing_WebApp/EldenRing_WebApp/Data/Scaling.cs
namespace EldenRing_WebApp.Data
{
    public class Scaling
    {
        public string name { get; set; }
        public string scaling { get; set; }

        public Scaling(string name, string scaling)
        {
            this.name = name;
            this.scaling = scaling;
        }
    }
}

File: EldenRing_API/demo/src/main/java/com/example/demo/Scaling.java
package com.example.demo;

import lombok.Data;


public class Scaling
{
    private String name;
    private String scaling;

    public Scaling(String name, String scaling) {
        this.name = name;
        this.scaling = scaling;
    }

    public String getName() {
        return name;
    }

    public String getScaling() {
        return scaling;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setScaling(String scaling) {
        this.scaling = scaling;
    }
}
